index.html

```diff
@@ -159,7 +159,56 @@ <h3 class="u-color-grey-dark">Social Media</h3>
     </div>
     <!-- Social Media container -->
 
-   
+       <!-- Testimonials -->
+       <section class="c-testimonials-section">
```

Comment:
Sve sekcije trebaju biti identične, što znači da imaju jednu zajedničku klasu, npr. `.c-section` i onda svaka sekcija ima svoju modifier klasu. Isto tako za elemente unutar sekcije kao što su naslov i podnaslo, to su sve elementi bloka `.c-section`.